src/components/Navigation/TopBar/TopBar.test.tsx

```diff
@@ -306,15 +306,9 @@ test('Renders with correct styles', () => {
     colors.ui.background__default.rgba
   );
   expect(topBar).toHaveStyleRule('padding-top', `8px`);
-  expect(topBar).toHaveStyleRule(
-    'padding-right',
-    `24px`
-  );
-  expect(topBar).toHaveStyleRule(
-    'padding-bottom',
-    `8px`
-  );
-  expect(topBar).toHaveStyleRule('padding-left', `${spacings.medium}`);
+  expect(topBar).toHaveStyleRule('padding-right', `24px`);
```

Comment:
This 24px and the 8px below could be the relevant token (large and small) instead of the string.